components/usage/pkg/db/billed_session_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package db_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gitpod-io/gitpod/usage/pkg/db"
+	"github.com/gitpod-io/gitpod/usage/pkg/db/dbtest"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBilledSession_WriteRead(t *testing.T) {
+	conn := dbtest.ConnectForTests(t)
+	teamID := uuid.New().String()
+	teamAttributionID := db.NewTeamAttributionID(teamID)
+
+	instance := dbtest.NewWorkspaceInstance(t, db.WorkspaceInstance{
```

Comment:
Why do we create a WorkspaceInstance record here?

Reply:
Now that you ask it I realize it's actually unnecessary 😬 Will remove it, thanks!